### DISCUSSION TITLE
OpenLane quit at synthesis stage but no apparent error

### DISCUSSION THREAD

**Discussion by heliboy2003** [The-OpenROAD-Project/OpenLane#1933] [Q&A]
Dear community,

I am pretty new to OpenLane, just started to try it out for only few days. I've successfully run the 'test' and one of my own design (very small) and able to generate GDS. But when I tried a larger design, OpenLane stop at Sythesis stage (Step 1) with no apparent error. The following is the message OpenLane generated in console

-----------------------------------------------------------------------------
root@Ubuntu20:/openlane# ./flow.tcl -design Proj
OpenLane 4c74d8b13c39ac20acfd412d90fdd689a959fafc
All rights reserved. (c) 2020-2022 Efabless Corporation and contributors.
Available under the Apache License, version 2.0. See the LICENSE file for more details.

[INFO]: Using configuration in 'Proj/config.json'...
[INFO]: PDK Root: /root/.volare
[INFO]: Process Design Kit: sky130A
[INFO]: Standard Cell Library: sky130_fd_sc_hd
[INFO]: Optimization Standard Cell Library: sky130_fd_sc_hd
[INFO]: Run Directory: /openlane/Proj/runs/RUN_2023.08.07_06.51.31
[INFO]: Saving runtime environment...
[INFO]: Preparing LEF files for the nom corner...
[INFO]: Preparing LEF files for the min corner...
[INFO]: Preparing LEF files for the max corner...
[INFO]: Running linter (Verilator) (log: Proj/runs/RUN_2023.08.07_06.51.31/logs/synthesis/linter.log)...
[INFO]: 0 errors found by linter
[WARNING]: 1149 warnings found by linter
[STEP 1]
[INFO]: Running Synthesis (log: Proj/runs/RUN_2023.08.07_06.51.31/logs/synthesis/1-synthesis.log)...
[ERROR]: during executing yosys script /openlane/scripts/yosys/synth.tcl
[ERROR]: Log: Proj/runs/RUN_2023.08.07_06.51.31/logs/synthesis/1-synthesis.log
[ERROR]: Last 10 lines:

[INFO]: USING STRATEGY AREA 0

64. Executing ABC pass (technology mapping using ABC).

64.1. Extracting gate netlist of module `\TxEncode' to `/tmp/yosys-abc-KdQOSV/input.blif'..
Extracted 182146 gates and 188238 wires to a netlist network with 6090 inputs and 4884 outputs.

64.1.1. Executing ABC.
child killed: kill signal

[ERROR]: Creating issue reproducible...
----------------------------------------------------------------------------------------------------

I've looked at the runs folder, found no meaningful message telling me what is wrong. So what I should do to resolve this situation?

Thanks

**Comment by heliboy2003** [comment]
After several attempts, the situation is not solved. I submit an issue and close this.

**Comment by krupatishbi** [comment]
has anyone solved this? I am getting the same error when trying to run my own design. it seems to also have "-errorcode NONE" and no matter what i try, there is no way to move past the ABC error.

[issue_reproducible.zip](https://github.com/user-attachments/files/16997209/issue_reproducible.zip)
